platform-sdk/swirlds-virtualmap/src/main/java/com/swirlds/virtualmap/internal/cache/VirtualNodeCache.java

```diff
@@ -1827,10 +1628,6 @@ private static final class Mutation<K, V> {
             this.version = version;
         }
 
-        byte getFlags() {
-            return flags;
-        }
-
         static boolean getFlag(final byte flags, final int bit) {
```

Comment:
This wasn't reverted, but I think it doesn't matter

Reply:
Agree